### ISSUE TITLE
[BUG] Korean is broken when requested by /sse api

### ISSUE THREAD

**Issue by shinsunyoung** [jobrunr/jobrunr#325]
**Are you running the latest version of JobRunr?**
Yes
If no, please first upgrade to the latest JobRunr version and check whether the issue still exists.

**Describe the bug**
I added a log, but the Korean is broken. It comes out well when requested by `/api`, but Korean is broken when requested by `/sse`.

I looked for code to contribute, but I couldn't find encoding-related settings. 😭

![image](https://user-images.githubusercontent.com/42836576/149915334-f62c5851-9a00-4973-9af2-159893773c22.png)
After calling `/sse`

**Environment**
I'm using JobRunr version: `4.0.6`
I'm running on JRE / JDK (e.g. OpenJDK 1.8.0_292): `11`
I'm using the following StorageProvider (e.g. Oracle / MongoDB / ...): `Redis(AWS Elastic Cache)`

**How to Reproduce**
Provide a simple Github project or code fragment which shows how to reproduce the issue (ideally a Github repo).
```java
private final JobRunrDashboardLogger jobRunrLogger = new JobRunrDashboardLogger(LoggerFactory.getLogger(PushEndpointService.class));

// ...

  @Job(name = "Send push(Endpoint)")
  public void push(SendPushPublishRequest request) {
    jobRunrLogger.info(request.toString()); // The headline and title of this object are Korean.

    // ...
  }


```

**Expected behavior**
Even after calling `/sse`, the log is displayed in Korean.

![image](https://user-images.githubusercontent.com/42836576/149916790-05a494ef-7d9a-404e-827d-1bf9b537dc59.png)

**Additional context**
N/A

**Comment by rdehuyss** [comment]
Feel free to create a PR, my time is limited for the moment.

**Comment by starrybleu** [comment]
The reason is that `SseExchange` creates `BufferedWriter` with `OutputStreamWriter` along with JVM default charset which is not corresponding to your encoded string.

Maybe you can solve this problem by passing `-Dfile.encoding=UTF-8(or any other proper encoding name)` to your JVM launching application

**Comment by rdehuyss** [comment]
Solved in v5.0.0 branch.
